examples/browser-webrtc/README.md

```diff
@@ -1,22 +1,43 @@
-# Rust-Libp2p Browser-Server WebRTC Example
-
-This example demonstrates how to use the `rust-libp2p` library in a browser.
-It uses [Leptos](https://leptos.dev/) WebAssembly framework to display the pings from the server.
-
-## Running the example
-
-First, start the `/server`:
-
-```sh
-cd server
-cargo run
-```
-
-Then, start the Leptos `/client`:
-
-```sh
-cd client
-trunk serve --open
-```
-
-This will open the browser where you will see the browser pinging the server. Open the server console logs to see the server pinging the browser.
+# Rust-libp2p Browser-Server WebRTC Example
+
+This example demonstrates how to use the `libp2p-webrtc-websys` transport library in a browser to ping the WebRTC Server.
+It uses [wasm-pack](https://rustwasm.github.io/docs/wasm-pack/) to build the project for use in the browser.
+The resulting `.js` bindings and `.wasm` can be served by any static web server, such as [`http-server`](https://github.com/http-party/http-server).
+
+## Running the example
+
+### WebRTC Server
+
+First, start the [`/server`](./server) in one terminal:
+
+```sh
+cd server
+cargo run
+```
+
+### WebRTC Client
+
+Then, start the client [`/client`](./client) in a separate terminal:
+
+```sh
+cd client
+wasm-pack build --target web
+```
+
+### Serve webpage
+
+Then serve the `index.html` with a static web server, such as [`http-server`](https://github.com/http-party/http-server):
+
+Using [npx](https://docs.npmjs.com/cli/v9/commands/npx) to run `http-server` without installing it globally, in another terminal, run:
+
+```bash
+npx http-server
+```
+
+or having done `npm install --global http-server`, simply:
+
+```bash
+http-server
+```
```

Comment:
We already depend on `axum` right? How difficult would it be to serve the static assets using that to avoid users have to download another component?

Reply:
It can be done, we'd have to write a simple static server for it though as seen here: https://github.com/tokio-rs/axum/discussions/446

Reply:
Implemented using `axum` static file server

Note that the server is in a different folder than the client, so there is a bit of cross-folder crate interaction but I don't see that as a big issue